update: Fix information on Android distributions page (#2844)
privacyguides/privacyguides.org: docs/android/distributions.md

---

> meta_title: "The Best Custom Android OS's - Privacy Guides"

Reviewer: ```suggestion
meta_title: "The Best Custom Android Operating Systems - Privacy Guides"
```

- [:material-target-account: Targeted Attacks](../basics/common-threats.md#attacks-against-specific-individuals){ .pg-red }
- [:material-bug-outline: Passive Attacks](../basics/common-threats.md#security-and-privacy){ .pg-orange }

> A **custom Android-based operating system** (often mistakenly called a ROM) can be a way to achieve a higher level of privacy and security on your device. This is in contrast to the "stock" version of Android which comes with your phone from the factory, and is often deeply integrated with Google Play Services as well as other vendor software.

Reviewer: ```suggestion
A **custom Android-based operating system** (sometimes referred to as a **custom ROM**) can be a way to achieve a higher level of privacy and security on your device. This is in contrast to the "stock" version of Android which comes with your phone from the factory, and is often deeply integrated with Google Play Services as well as other vendor software.
```

Reviewer: FYI the original wording was based on https://en.wikipedia.org/wiki/List_of_custom_Android_distributions

> [Android](https://en.wikipedia.org/wiki/Android_(operating_system))-based [operating systems](https://en.wikipedia.org/wiki/Operating_system) (OS) commonly referred to as Custom ROMs or Android ROMs

> We recommend installing GrapheneOS if you have a Google Pixel as it has improved security hardening and privacy features. The reason we don't list other devices or OSes is because they often have:

Reviewer 2: ```suggestion
We recommend installing GrapheneOS if you have a Google Pixel as it has improved security hardening and privacy features. The reasons we don't list other devices or OSes are as follows:
```

> We recommend installing GrapheneOS if you have a Google Pixel as it provides improved security hardening and additional privacy features. The reasons we don't list other OS's or devices are as follows:

Reviewer: ```suggestion
We recommend installing GrapheneOS if you have a Google Pixel as it provides improved security hardening and additional privacy features. The reasons we don't list other operating systems or devices are as follows:
```

> - [weaker security](index.md#install-a-custom-distribution)
> - may simply drop support when the maintainer loses interest or upgrades their device as opposed to GrapheneOS which has a predictable [support cycle](https://grapheneos.org/faq#device-lifetime).
> - few or no notable privacy or security features that make the move worthwhile

Reviewer 2: ```suggestion
- These devices and/or OSes often have [weaker security](index.md#install-a-custom-distribution).
- Support for these devices and/or OSes may simply be dropped when the maintainer loses interest or upgrades their device, which is in contrast to the predictable [support cycle](https://grapheneos.org/faq#device-lifetime) that GrapheneOS follows.
- These OSes simply have few or no notable privacy or security features that make installing them worthwhile.
```

Author: Seems like the plural of OS should be OS’s
https://www.grammar-monster.com/lessons/abbreviations_forming_plurals.htm

> - These OS's and the devices on which they're installed often have [weaker security](index.md#install-a-custom-distribution).
> - Support for these OS's and/or devices may simply be dropped when the maintainer loses interest or upgrades their device, which is in contrast to the predictable [support cycle](https://grapheneos.org/faq#device-lifetime) that GrapheneOS follows.
> - These OS's simply have few or no notable privacy or security features that make installing them worthwhile.

Reviewer: ```suggestion
- They often have [weaker security](index.md#install-a-custom-distribution).
- Support is frequently dropped when the maintainer loses interest or upgrades their device, which is in contrast to the predictable [support cycle](https://grapheneos.org/faq#device-lifetime) that GrapheneOS follows.
- They generally have few or no notable privacy or security improvements that make installing them worthwhile.
```

- [:material-target-account: Targeted Attacks](../basics/common-threats.md#attacks-against-specific-individuals){ .pg-red }
- [:material-bug-outline: Passive Attacks](../basics/common-threats.md#security-and-privacy){ .pg-orange }

A **custom Android-based operating system** (often mistakenly called a ROM) can be a way to achieve a higher level of privacy and security on your device. This is in contrast to the "stock" version of Android which comes with your phone from the factory, and is often deeply integrated with Google Play Services as well as other vendor software.

We recommend installing GrapheneOS if you have a Google Pixel as it has improved security hardening and privacy features. The reason we don't list other devices or OSes is because they often have:

> - [weaker security]{android.md#install-a-custom-distribution}

Author: Fixed
